packages/devtools-reps/src/object-inspector/tests/component/events.js

```diff
@@ -39,18 +44,15 @@ describe("ObjectInspector - properties", () => {
     const stub = gripRepStubs.get("testMaxProps");
     const onDoubleClick = jest.fn();
 
-    const oi = mount(ObjectInspector({
-      autoExpandDepth: 0,
+    const oi = mount(ObjectInspector(generateDefaults({
       roots: [{
         path: "root",
         contents: {
           value: stub
         }
       }],
-      getObjectProperties: () => {},
-      loadObjectProperties: () => {},
       onDoubleClick,
-    }));
+    })));
 
     const node = oi.find(".node").first();
     node.simulate("doubleclick");
```

Comment:
this is a side comment, but the fact that we need to break up the component test by entries, classnames, events, etc makes me worry about test readability long term...

I think some architecture like redux and util tests would help so we can take some pressure off these component tests.